EventListener/RouteAnnotationEventListener.php

```diff
@@ -75,7 +82,11 @@ public function registerRouteAnnotation(SitemapPopulateEvent $event)
         $section = $event->getSection();
 
         if (is_null($section) || $section === $this->defaultSection) {
-            $this->addUrlsFromRoutes($event);
+            if ($this->alternateSection) {
```

Comment:
This is too much duplicated code for me, we should just be able to decorate the URL in `addUrlsFromRoutes` this will avoid duplicating the whole logic in two separated methods

Reply:
Removed method to prevent duplicate code